main/webapp/modules/core/scripts/reconciliation/recon-dialog.html

```diff
@@ -12,6 +12,8 @@
 				</div>
 				<div class="recon-dialog-service-panel-container"
 					bind="servicePanelContainer">
+					<div class="recon-dialog-service-panel-message-note"
+						bind="servicePanelMessageNote"></div>
```

Comment:
When using this dialog, identifiers are indeed validated, so we should not add this note in this dialog. The operation which does not validate its identifiers is the "Use values as identifiers" operation.

Reply:
How did I miss that, and its clearly stated on how to reproduce the issue . Working on it.

> Thanks! It looks like your PR is still deleting two unrelated files (in `server/IDEs/eclipse`).

Will try and figure out how those files are generated. It could be my eclipse ide